lib/classes/Swift/FileSpool.php

```diff
@@ -165,10 +165,13 @@ public function flushQueue(Swift_Transport $transport, &$failedRecipients = null
             /* We try a rename, it's an atomic operation, and avoid locking the file */
             if (rename($file, $file.'.sending')) {
                 $message = unserialize(file_get_contents($file.'.sending'));
+                if ($message !== false) {
```

Comment:
I would throw an exception instead.

Reply:
@fabpot Wouldn't that cause the same problem, namely a fatal error? And thus causing all subsequent mails to not be sent. That seems not the desired behaviour.